Jenkinsfile.github

```diff
@@ -222,13 +223,13 @@ pipeline {
                         env.SNYK_RESULTS_SHEET = "${env.RELEASE_NAME}-${env.RELEASE_VERSION}-snyk-results.xlsx"
                         env.SNYK_RESULTS_SHEET_URL = "${env.RELEASE_BASEURL}/${env.SNYK_RESULTS_SHEET}"
                         slackSend(channel: env.SLACK_CHANNEL_NOTIFY, color: "good", message: "<${env.BUILD_URL}|CSM ${env.RELEASE_VERSION}> - :white_check_mark: Success!\n- Release distribution: <${env.RELEASE_URL}|${env.RELEASE_NAME}-${env.RELEASE_VERSION}.tar.gz>\n- Snyk results: <${env.SNYK_RESULTS_SHEET_URL}|${env.SNYK_RESULTS_SHEET}> (raw scan results: <${env.SNYK_RESULTS_URL}|${env.SNYK_RESULTS_FILENAME}>)")
-                        build(job: jenkinsUtils.findJob("Cray-HPE", "csm-release-internal-upload", env.CSM_VSHASTA_DEPLOY_BRANCH).getFullName(), wait: false, parameters: [
+                        build(job: "Cray-HPE/csm-release-internal-upload/main", wait: false, parameters: [
```

Comment:
@phalseth-hpe I set `main` branch here - I don;t see any other branches in `csm-release-internal-upload`.
Please feel free to backport new set of parameters (https://github.com/Cray-HPE/csm/commit/9e79ae246289555b941b8a484d80baaf629debd8) here. BTW, if default values for `POSTUPLOAD_REPORT` is `always`, you can omit it.